R/k-NCN_curve_functions.R

```diff
@@ -0,0 +1,87 @@
+# Internal function for kNCN_average (). k-NCN algorithm starting with a specified focal sample
+
+centroid_accumulate<-function(x,focal_sample= 1, n=NULL, coords=NULL){
```

Comment:
This function accumulates starting with a specified focal sample. It is the internal working horse for `kNCN_average()` that generates the average curves across all possible start samples.